### ISSUE TITLE
Request for a bigger mouse pointer in videos

### ISSUE THREAD

**Issue by mazenbesher** [koaning/calmcode-feedback#197]
I have a small suggestion. The mouse pointer used in the videos is small and can be hard to follow, especially when demonstrating code (the text pointer is hard to follow!). A larger and more visible mouse pointer in the videos would be a great help.

**Comment by koaning** [comment]
Do you have a specific video where this is most painful as an example? 

I try to rely less and less on the mouse pointer and instead to use the "drawing over the screen"-trick as much as possible. I certainly didn't always do that, but the recent videos should show it more and more.

**Comment by mazenbesher** [comment]
There isn't a specific video where this issue is particularly apparent. However, while watching the recent Playwright series, I found myself trying to locate the cursor a couple of times, such as when you were highlighting imported functions. It's not a major issue, and you're doing a great job marking important details. However, I find myself naturally inclined to follow the pointer.

**Comment by koaning** [comment]
I'll see if there's a nicer way to handle this. Will close the issue for now though since it's not a specific bug on the current site.
